src/views/PurchaseOrderDetail.vue

```diff
@@ -18,23 +18,24 @@
     <ion-content :fullscreen="true">
       <div>
         <ion-item lines="none">
-          <h1>{{$t("Purchase Order")}}: PO10291</h1>
+          <h1>{{$t("Purchase Order")}}: {{ orderDetail[0]?.externalOrderId }}</h1>
         </ion-item>
         <ion-item>
           <ion-label>{{$t("Scan Items")}}</ion-label>
           <ion-label>
             <ion-input placeholder="Scan barcodes to receive" />
           </ion-label>
         </ion-item>
-        <ion-card>
+        <ion-card v-for="(item, index) in orderDetail" :key="index">
           <div class="product-info">
             <ion-item lines="none">
               <ion-thumbnail slot="start">
-                <img src="https://cdn.shopify.com/s/files/1/0069/7384/9727/products/test-track.jpg?v=1626255137" />
+                <!-- TODO:- handle this getter when another pull request get merged -->
+                <img src="getProduct(item.productId).mainImageUrl" />
               </ion-thumbnail>
               <ion-label>
-                <ion-label>Chaz Kangeroo Hoodie-XS-Green</ion-label>
-                <p>12203</p>
+                <ion-label>{{ item.productName }}</ion-label>
```

Comment:
Use product getter to display product-related information.

Reply:
Changed implementation Sir.